purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -46,6 +49,7 @@ import java.net.URL
  */
 class Purchases internal constructor(
     @get:JvmSynthetic internal val purchasesOrchestrator: PurchasesOrchestrator,
+    @get:JvmSynthetic internal val virtualCurrencyManager: VirtualCurrencyManager,
```

Comment:
Hmm could we move this dependency to the `PurchasesOrchestrator`? I would consider `Purchases` just a facade, without any real logic, that just calls `PurchasesOrchestrator` as needed. Then, in CEC we just don't add the API methods calling `PurchasesOrchestrator`.

Reply:
I think on iOS we've been a little less thorough in keeping everything inside PurchasesOrchestrator, and we do have some logic in the Purchases class 😅

https://github.com/RevenueCat/purchases-ios/blob/main/Sources/Purchasing/Purchases/Purchases.swift#L254-L281